hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/BlockManagerImpl.java

```diff
@@ -149,20 +148,19 @@ public void stop() throws IOException {
   /**
    * Allocates a block in a container and returns that info.
    *
-   * @param size - Block Size
-   * @param type Replication Type
-   * @param factor - Replication Factor
```

Comment:
NIT: Add in the param for ReplicationConfig (also owner is missing too, but not due to this change, so may as well add it in).

Reply:
:+1: